build.sbt

```diff
@@ -33,7 +33,6 @@ lazy val baseSettings = Seq(
   crossScalaVersions := Seq(scala213, scala3),
   organization := "typify",
   version := "12.0.0",
-  resolvers += "bondlink-maven-repo" at "https://raw.githubusercontent.com/mblink/maven-repo/main",
```

Comment:
We don't want to remove this, we should update it to the new S3 URL, `https://s3.amazonaws.com/bondlink-maven-repo`

It's necessary for `mimaReportBinaryIssues` to work so it can download previous versions of the lib

Reply:
That makes sense, updated.